test/index.js

```diff
@@ -648,6 +648,17 @@ describe('Nightmare', function () {
       var stats = fs.statSync(tmp_dir+'/test2.pdf');
       stats.size.should.be.at.least(1000);
     });
+
+    it('should return a buffer from a PDF with no path', function*() {
+      var buf = yield nightmare
+        .goto(fixture('manipulation'))
+        .pdf({printBackground: false});
+
+      var isBuffer = buf instanceof Buffer;
```

Comment:
Yeah good point.

Although the underlying implementation is the same (see: https://github.com/nodejs/node/blob/master/lib/buffer.js#L180), it might be worth changing for readability.

Thoughts?